src/webhooks/options-automator/options-automator.ts

```diff
@@ -26,15 +26,49 @@ export async function sendOptionAutomatorUpdatesToDataDog(
   message: OptionsAutomatorResponse,
   timestamp: number
 ) {
-  // Not sure how detailed we want the text message to be, but this should be fine initially
-  const params: v1.EventCreateRequest = {
-    title: 'Options Automator Update',
-    text: JSON.stringify(message),
-    alertType: 'info',
-    dateHappened: timestamp,
-    tags: [`sentry_region:${message.region}`],
+  const formatRegionTag = (region: string): string => {
+    if (region === 'us' || region === 'de') {
```

Comment:
Can you separate this out into an const variable array of `SAAS_REGIONS` or something? Just so it makes it a bit easier to update in the future when we do add more regions

That would look something like
```
SAAS_REGIONS = ['us', 'de'];
...
if (SAAS_REGIONS.includes(region)) {...
```